src/website/src/components/Navigation/Navigation.jsx

```diff
@@ -36,12 +36,13 @@ const Navigation = React.memo(
           sidebarOpen ? '' : ` ${styles['nav--closed']}`
         }${forceRefresh ? ' ' : ''}`}
       >
-        <div className={styles.header}>
+        <header className={styles.header}>
+          <span className={styles.version}>v2.1.0</span>
```

Comment:
Can't we do better than hard-coding?
This creates a liability, we will have to update it manually every single time we make a release.
lerna.json contains the current version number.